### ISSUE TITLE
Can't load campaign file 47.1

### ISSUE THREAD

**Issue by Rince-Wind** [MegaMek/mekhq#1342]
### Environment
What version of MekHQ does your issue apply to?
47.1
Windows 7
Java 8, update 131

### Description
As above, I can't load the save.

[Artemis' Hunters30800916.zip](https://github.com/MegaMek/mekhq/files/3853216/Artemis.Hunters30800916.zip)
[megameklog.txt](https://github.com/MegaMek/mekhq/files/3853219/megameklog.txt)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/3853220/mekhqlog.txt)

**Comment by HammerGS** [comment]
From the log
```
java.lang.NullPointerException
	at mekhq.campaign.io.CampaignXmlParser.processPartNodes(CampaignXmlParser.java:1710)
	at mekhq.campaign.io.CampaignXmlParser.parse(CampaignXmlParser.java:271)
	at mekhq.campaign.CampaignFactory.createCampaign(CampaignFactory.java:88)
	at mekhq.gui.dialog.DataLoadingDialog$Task.doInBackground(DataLoadingDialog.java:210)
	at mekhq.gui.dialog.DataLoadingDialog$Task.doInBackground(DataLoadingDialog.java:129)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
```
Relates to #1332 and #1333 which are fixed in #1335 and in the pending 0.47.2 release

**Comment by Rince-Wind** [comment]
After reading through the related issues I found the offending vee and deleted it and everything with its unit id and I can load the save again.
Thanks for pointing me to it.
